static/templates/recent_topic_row.hbs

```diff
@@ -0,0 +1,29 @@
+{{!--
+Empty topics are used to add topics
+before and after a topic. This avoids
+a full rerender of the UI.
+--}}
+<tr class="empty_topic"></tr>
```

Comment:
UI part of inplace logic.

Reply:
I changed the to approach to using before and after jquery function as they are cleary better.